common/laws/00_bureaucracy.txt

```diff
@@ -206,9 +206,9 @@ law_meritocratic_bureaucracy = {
 	icon = "gfx/interface/icons/law_icons/meritocratic_bureaucracy.dds"
 	progressiveness = 0
 	modifier = {
-		interest_group_ig_intelligentsia_pol_str_mult = 0.1
+		interest_group_ig_intelligentsia_pop_attraction_mult = 0.1
 		state_tax_capacity_mult = 0.25
-		country_government_wages_mult = 0.10
+		country_government_wages_mult = 0.15
```

Comment:
mmm... government wages have unintended consequences. So I hope you know that this is safe, and doesn't cripple a nations by a positive wage feedback loop. (especially in combination with high government wages in budget).

Reply:
This effectively makes it on par with increasing your government wages by one level. It basically adds a cost to this law for being otherwise extremely favorable, in that you're always running at one higher tick of government wages. As far as my testing has been, this did not do any significant harm or wage feedback with 10% so I don't see why a round 15% would either and this law is already too powerful.